src/cg_ftoc.c

```diff
@@ -2546,6 +2546,490 @@ CGNSDLL void cg_1to1_average_write_f(
 					  *AverageInterfaceType);
 }
 
+/* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - *\
+ *   Read and write ParticleZone_t Nodes                                 *
+\* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
+
```

Comment:
FYI, we are moving away from "C wappers" for Fortran and instead calling the C APIs directly from Fortran (cgns_f.F90). If it is easy for you to do that, it is preferred; otherwise, I can do it. I don't want it to hold up the PR.

I need to put a developer's note in cg_ftoc.c noting this policy,